plugins/single_plugins/idle.cpp

```diff
@@ -136,15 +138,20 @@ class wayfire_idle
         }
     }
 
-    void idle_inhibit()
+    void idle_inhibit(bool because_fullscreen)
```

Comment:
why do we need this flag at all ???

Reply:
We need to track `idle_inhibited_by_fullscreen` to make sure that `disable_on_fullscreen_changed` doesn't mess with the protocol-based inhibitors.